Makefile

```diff
@@ -185,6 +185,7 @@ build/compiler-rt.BUILT: build/llvm.BUILT
 # $(3): the name of the target being built for
 # $(4): extra compiler flags to pass
 LIBCXX_CMAKE_FLAGS = \
+    -DNDEBUG=1 \
```

Comment:
Is `-DCMAKE_BUILD_TYPE=RelWithDebInfo` below not enough to get this defined?

Reply:
Yeah, from further inspection it looks like it. This flag isn't piped through to `make`. See the thread for why I included it in the first place.